src/utils/io.py

```diff
@@ -13,11 +13,19 @@ class DisplayConfig:
 
 class Format:
     @staticmethod
-    def colored(text: str, color: str) -> str:
+    def colored(text: str, color: str, style: str = 'normal') -> str:
```

Comment:
Maybe as you add a style and not only a color, I would replaced the function name by `styled()`